R/fotmob_stats.R

```diff
@@ -39,19 +39,35 @@
   x
 }
 
+
+.extract_seasons_and_stats_from_options <- function(options) {
```

Comment:
this logic can be used in 2 of the clauses of the if-else in the `.fotmob_get_stat_and_season_options` function, so it made sense to put it into its own function

Reply:
Awesome